rest/taskana-rest-spring/src/main/java/pro/taskana/resource/rest/AbstractRessourcesAssembler.java

```diff
@@ -1,68 +0,0 @@
-package pro.taskana.resource.rest;
-
-import java.util.Map;
-import javax.servlet.http.HttpServletRequest;
-import org.springframework.hateoas.Link;
-import org.springframework.hateoas.PagedModel.PageMetadata;
-import org.springframework.web.context.request.RequestContextHolder;
-import org.springframework.web.context.request.ServletRequestAttributes;
-import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
-import org.springframework.web.util.UriComponentsBuilder;
-
-import pro.taskana.common.rest.models.PagedResources;
-
-/**
- * Abstract resources assembler for taskana REST controller with pageable resources. This method is
- * deprecated, it can be removed after fixing taskana-simple-history references
- */
-// TODO: @Deprecated
-// TODO: please remove spring-webmvc dependency
```

Comment:
please also remove this dependency ;)

Reply:
Won't fix due to "import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;" dependency in "TaskanaRestExceptionHandler"

Reply:
Oh. missed that. thanks!